jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MavenProjectProperties.java

```diff
@@ -187,8 +201,10 @@ public JibContainerBuilder createJibContainerBuilder(
       throws IOException {
     try {
       if (isWarProject()) {
-        Path explodedWarPath =
-            Paths.get(project.getBuild().getDirectory(), project.getBuild().getFinalName());
+        Build build = project.getBuild();
+        Path war = Paths.get(build.getDirectory(), build.getFinalName() + ".war");
+        Path explodedWarPath = tempDirectoryProvider.newDirectory();
+        ZipUtil.unzip(war, explodedWarPath);
```

Comment:
Sort of unrelated to this PR: I haven't tested, but I noticed that the unzip logic is very similar to the tar extraction logic. I don't really know anything about the zip file format, but I wonder if the fix [here](https://github.com/GoogleContainerTools/jib/pull/2068) should also be applied to `ZipUtil.unzip()`.

Reply:
I think that is a very good idea.